### PR TITLE
Converts reagent containers to Initialize, fixes hand-chew cool down and moves suit sensor Z-level list

### LINKED ISSUE #3361: Crew monitoring's Z-levels are hard-coded in a list in an obscure file.

### LINKED ISSUE #3362: Reagent containers should be converted to Initialize().

### LINKED ISSUE #3363: Hand chewing uses a global cooldown instead of a per-mob/per-client cooldown.

### PR DESCRIPTION
**Pull request by printer16**
Converts all reagent containers to use Initialize, fixes the hand-chew cool down so it's no longer for everyone, and moves the suit sensor Z-level to the global.dm right above the station name so anyone else using the code can't miss it.

Closes #3363 
Closes #3361 
Closes #3362

### CODE REVIEW

File: code/game/machinery/kitchen/icecream.dm [hunk 1]
```diff
@@ -178,7 +178,8 @@
 	var/ice_creamed = 0
 	var/cone_type
 
-/obj/item/weapon/reagent_containers/food/snacks/icecream/New()
+/obj/item/weapon/reagent_containers/food/snacks/icecream/Initialize()
+	. = ..
```
Comment (Lohikar) [inline code/game/machinery/kitchen/icecream.dm]: Bad return.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -4493,7 +4490,7 @@
 	icon_state = "koisjelly"
 	filling_color = "#E6E600"
 
-/obj/item/weapon/reagent_containers/food/snacks/koisjelly/New()
+/obj/item/weapon/reagent_containers/food/snacks/koisjelly/Initialize()
 	..()
```
Comment (Lohikar) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Missing parent return.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -58,9 +58,9 @@
 	amount_per_transfer_from_this = 5
 	volume = 5
 
-/obj/item/weapon/reagent_containers/hypospray/autoinjector/New()
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/Initialize()
 	..()
-	reagents.add_reagent("inaprovaline", 5)
+	. = reagents.add_reagent("inaprovaline", 5)
```
Comment (Lohikar) [inline code/modules/reagents/reagent_containers/hypospray.dm]: wat

File: code/game/machinery/kitchen/icecream.dm (hunk 1 above)
Comment (printer16) [reply code/game/machinery/kitchen/icecream.dm]: Read your mind and fixed it before you made the review

Comment (Lohikar) [reply code/game/machinery/kitchen/icecream.dm]: 😠

**Comment by printer16** [comment]
Got the rest of them. Good for review now.

**Comment by Lohikar** [comment]
> 820 lines changed

o7

**Comment by skull132** [comment]
Everything, obviously.

**Pull request merged**